package.json

```diff
@@ -41,59 +42,72 @@
     ]
   },
   "dependencies": {
-    "@sambego/storybook-state": "^1.3.2",
+    "@babel/core": "^7.8.7",
```

Comment:
I think these Babel and Jest dependencies should be devDependencies

Reply:
I completely removed babel since we won't use it anymore. And put jest in dev dependencies